lib/Service/SmimeService.php

```diff
@@ -517,8 +551,49 @@ public function isEncrypted(Horde_Imap_Client_Data_Fetch $message): bool {
 		/** @var Horde_Mime_Headers_ContentParam_ContentType $contentType */
 		$contentType = $headers['content-type'];
 		return $contentType->ptype === 'application'
-			&& $contentType->stype === 'pkcs7-mime'
+			&& ($contentType->stype === 'pkcs7-mime' || $contentType->stype === 'x-pkcs7-mime')
```

Comment:
@st3iny 

please have a look at this

Thunderbird uses `pkcs7-mime`
openssl_pkcs7_encrypt uses `x-pkcs7-mime`

Reply:
Fine by me. It works with Thunderbird so I expect other clients to recognize this mime type too.

**PS:** I can't find any reference to this header in the RFCs.